docs/main/economic-viability.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Economic viability of power plants
+---
+
+This section explains the economic viability of power plants, as presented in the following tables:
+- **Economic viability of electricity production technologies**
+- **Economic viability of electricity storage technologies**
+- **Economic viability of flexible electricity demand technologies**
+
+As described in the [Cost methods](cost-methods.md), the ETM separates fuel costs from OPEX and CAPEX for each cost group. To accurately assess the economic viability of power production facilities, fuel costs must be included in the calculations.
```

Comment:
Yes, it is true that fuel costs must be included. I would however rather say that the 'normal' cost method use a systems approach, meaning that fuel and emissions costs are accounted on a system-level and in their own category. These tables use the assets approach.